packages/playground/asset-import/vite.config.js

```diff
@@ -0,0 +1,12 @@
+const svelte = require('@sveltejs/vite-plugin-svelte');
+const { defineConfig } = require('vite');
+
+module.exports = defineConfig(({ command, mode }) => {
+	const isProduction = mode === 'production';
+	return {
+		plugins: [svelte({ extensions: ['.svelte', '.html', '.svg'] })],
```

Comment:
at least in vite1, .html extension didn't work for svelte, vite treats that entirely different. Does it work now?

Reply:
Yeah, it seems to work. Though I should probably test with a more sophisticated Svelte code in the HTML to check if Vite is doing anything else.

Regarding the failing tests, I had issues getting Playwright to startup the browser so I couldn't actually test locally. So I'm sort of winging it for this PR.

Reply:
@dominikg Through some testing, the current change can handle HTML inports only in dev mode, but fails in build mode. Turns out setting `.html` extension would trigger the plugin to compile the entrypoint HTML as a Svelte component, which generates a final empty bundle.

I've removed HTML support for now, but there could be ways to support this in the future, like detecting HTML files to avoid Svelte compile via the `transformIndexHtml` hook, though a bit sketchy. Would need to figure out a more solid strategy for that.

I also fixed the tests.